src/components/panel/panel.spec.js

```diff
@@ -94,6 +94,84 @@ describe('$mdPanel', function() {
     $animate.enabled(true);
   });
 
+  describe('provider logic:', function() {
+    var config = {
+      name: 'testConfig',
+      options: DEFAULT_CONFIG
+    };
+    var config2 = {
+      name: 'testConfig2',
+      options: DEFAULT_CONFIG
+    };
+
+    it('should have the $mdPanelConfigProvider available',
+        inject(function($mdPanelConfig) {
```

Comment:
Tests shouldn't have inject within in. These should all be in a beforeEach at the top of the file.